apps/sms/services/js/message_manager.js

```diff
@@ -137,28 +187,39 @@ var MessageManager = {
     };
   },
 
+  /**
+   * Function to get the message object having a given id.
+   * @param {number} id The id of the message we wish to get.
+   */
   getMessage: function mm_getMsg(id) {
     return this._mozMobileMessage.getMessage(id);
   },
 
+  /**
+   * Function to get the MMS object having a given id.
```

Comment:
__nit:__ please take a look at mdn for the description here and in other places as well.